fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/metric/FiliMetricName.java

```diff
@@ -0,0 +1,41 @@
+package com.yahoo.wiki.webservice.data.config.metric;
+
+import com.yahoo.bard.webservice.data.config.names.ApiMetricName;
+import com.yahoo.bard.webservice.data.time.TimeGrain;
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Created by kevin on 3/7/2017.
+ */
+public class FiliMetricName implements ApiMetricName {
+    private String apiName;
+    private List<TimeGrain> satisfyingGrains;
+
+    public FiliMetricName(String lowerCaseName, List<TimeGrain> timeGrains) {
+        this.apiName = EnumUtils.camelCase(lowerCaseName);
```

Comment:
Javadoc.
Is it REALLY a lowercase name?
Should this be plural time grains?  This config currently only allows a single default grain.